### ISSUE TITLE
fix(docs): Improve the example in the docstring of `skore.cross_validate`

### ISSUE THREAD

**Issue by sylvaincom** [probabl-ai/skore#663]
The code snippet is not auto-contained

**Comment by augustebaum** [comment]
Looks like this can be closed, given the discussions in #678? If you do close it make sure to summarize why

**Comment by sylvaincom** [comment]
Yes, we can close it, we debated about it in #678 and this was not a strong opinion for me
